src/main/java/betterquesting/client/gui2/editors/GuiPrerequisiteEditor.java

```diff
@@ -228,6 +237,21 @@ private boolean containsReq(IQuest quest, int id) {
         return false;
     }
 
+    private void reorderReq(IQuest quest, int id) {
+        int[] orig = quest.getRequirements();
+
+        int indexToShift = -1;
+        for (int i = 0; i < orig.length; i++) if (orig[i] == id) indexToShift = i;
```

Comment:
Could you add some bracket formatting and separate these out onto different lines please? It is kinda hard for me to read when the entire statement is one line.